### ISSUE TITLE
Label with an icon

### ISSUE THREAD

**Issue by tothda** [clj-pdf/clj-pdf#204]
I'd like to add an icon to a piece of text. For instance to add a calendar icon in front of a date. 

At first I wanted to use an SVG from the FontAwesome library, but as I see SVGs are positioned relative to the top left corner of the page. Is there a simple way to solve this problem using SVG icons?

**Comment by yogthos** [comment]
There isn't a way to position SVGs relative to text at the moment. I'd be open to adding that as feature, but don't have time to look at it in the near future. I would be open to a PR if somebody has the time before I get to it.

**Comment by prestancedesign** [comment]
Hi, Seems to be still the case, can you confirm please?

**Comment by yogthos** [comment]
Yup, this is still an open issue.

**Comment by yogthos** [comment]
That's correct, there isn't support for positioning svg relative to the page at the moment.

**Comment by prestancedesign** [comment]
Thanks for the confirmation and this awesome lib!

**Comment by yogthos** [comment]
Thanks, and glad it's coming in handy. :)
